cspell.json

```diff
@@ -1,5 +1,7 @@
 {
-	"dictionaries": ["typescript"],
+	"dictionaries": [
+		"typescript"
+	],
```

Comment:
[Build] This is an example of the [Prettier](https://github.com/OpenContributionsProject/opencontributionssite/actions/runs/4974912688/jobs/8915595919?pr=107) task failing. You can fix this by running `pnpm format:write` locally.

Alternately, if you're editing in VS Code locally (or in a Codespace I believe) and have the suggested extensions installed, saving a file should auto-format with the Prettier for VS Code extension.

Also, if you run `pnpm install` locally (or in a Codespace), making a commit should auto-format it for you.

See [_`.github/DEVELOPMENT.md`_](https://github.com/OpenContributionsProject/opencontributionssite/blob/af52be90494491d733ff500ab41ba26404483ae4/.github/DEVELOPMENT.md).